tools/ci/azure/cleanup_win10.yml

```diff
@@ -50,7 +57,41 @@ steps:
         Restart-Computer -Force
       }
     }
+    if (($stableExitCode -ne $exitCodeSuccess) -and
+        ($canaryExitCode -ne $exitCodeSuccess) -and
```

Comment:
Nit: Order stable, beta, dev, canary would be nice, or perhaps alphabetical. Same above and below.

Reply:
Fixed